apps/aptos/views/Ifos/hooks/vesting/useFetchUserWalletIfoData.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable camelcase */
+import { BIG_ZERO } from '@pancakeswap/utils/bigNumber'
+import BigNumber from 'bignumber.js'
+import { Ifo, PoolIds } from 'config/constants/types'
+import { useCallback } from 'react'
+import { IFO_RESOURCE_ACCOUNT_TYPE_METADATA } from 'views/Ifos/constants'
+import { VestingCharacteristics } from 'views/Ifos/types'
+import { useIfoPool } from '../useIfoPool'
+import { useIfoResources } from '../useIfoResources'
+import { useVestingCharacteristics } from './useVestingCharacteristics'
+
+export interface VestingData {
+  ifo: Ifo
+  userVestingData: {
+    vestingStartTime: number
+    [PoolIds.poolUnlimited]: VestingCharacteristics
+  }
+}
+
+export const useFetchUserWalletIfoData = () => {
+  const resources = useIfoResources()
+  const pool = useIfoPool()
+  const vestingCharacteristics = useVestingCharacteristics()
```

Comment:
seems like we need to refetch these three hooks on `fetchUserWalletIfoData`
can get the `refetch` and await on below callback
`const { data: pool, refetch: refetchPool } = useIfoPool();`